release/release.go

```diff
@@ -151,6 +151,32 @@ func CheckUpstreamRelease(ctx context.Context, client *github.Client, org, repo
 	return releases, nil
 }
 
+func KubernetesGoVersion(ctx context.Context, client *github.Client, version string) (string, error) {
+	var githubError *github.ErrorResponse
+
+	file, _, _, err := client.Repositories.GetContents(ctx, "kubernetes", "kubernetes", ".go-version", &github.RepositoryContentGetOptions{
+		Ref: version,
+	})
+
+	if err != nil {
+		if errors.As(err, &githubError) {
```

Comment:
What was the reason in choosing `errors.As` versus `errors.Is`?

Reply:
Because  As()  is useful when you have an error that has been encapsulated in another error and you want to access the value of the original error. What you think ?

Reply:
Awesome. That's what I was looking for.